tests/e2e/specs/e2e.spec.ts

```diff
@@ -189,6 +189,13 @@ test("body attributes are copied over", async ({ page }) => {
   expect(await body.getAttribute("data-other")).toEqual("false");
 });
 
+test("uses config body attributes", async ({ page }) => {
+  await page.goto("./vite-page");
+  const body = page.locator("body").last();
+  expect(await body.getAttribute("id")).toEqual("test-id");
+  expect(await body.getAttribute("class")).toEqual("test-classname");
```

Comment:
it would be ideal to navigate to a diff page with diff body attrs and verify they change. That way we know onrenderclient is working